src/DistributedObjectEvents.md

```diff
@@ -115,25 +115,10 @@ Entry Removed:EntryEvent{entryEventType=REMOVED, member=Member [192.168.1.100]]:
 ![image](images/NoteSmall.jpg) ***NOTE:*** *Please note that the method `IMap.clear()` does not fire an "EntryRemoved" event, but fires a "MapCleared" event.*
 <br></br>
 
+<br></br>
+![image](images/NoteSmall.jpg) ***NOTE:*** *A listener is not allowed to block a thread. Blocking operations have to be offloaded to another thread (pool).*
```

Comment:
is the passive voice beneficial here? "Listeners have to offload all blocking operations to another thread (pool)"

Reply:
👍 , let's make it active as you suggested.